Vagrantfile

```diff
@@ -129,6 +134,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
 
     app.vm.provision "ansible" do |ansible|
+      ansible.extra_vars = extra_vars
```

Comment:
Instead of using `extra_vars` here, I think that we should use the `lookup` template function to pull variables from the environment. Some examples of that technique exists in `group_vars/all`.